client/package.json

```diff
@@ -67,8 +67,8 @@
     "@storybook/addon-links": "^5.3.19",
     "@storybook/addons": "^5.3.19",
     "@storybook/react": "^5.3.19",
+    "@testing-library/dom": "^7.22.2",
     "@testing-library/jest-dom": "^5.10.0",
-    "@testing-library/react": "^10.0.6",
```

Comment:
We shouldn't remove this.

Reply:
thank you @julianguyen: just made the edits https://github.com/AudreyKj/ifme/commit/500bd7748185bdd97f4f6fdc86bfca7abe5597c5